peps/converters.py

```diff
@@ -181,7 +181,14 @@ def convert_pep_page(pep_number, content):
 
     data['content'] = str(pep_content)
 
-    source_link = "https://github.com/python/peps/blob/master/pep-{0}.txt".format(pep_number)
+    pep_ext = '.txt'
+    pep_rst_source = os.path.join(settings.PEP_REPO_PATH,
+                            'pep-{}.rst'.format(pep_number))
+    if os.path.exists(pep_rst_source):
+        pep_ext = '.rst'
```

Comment:
That's an OK solution for just `.txt` or `.rst`.
Wouldn't it be possible to write a generic test, by looking for a glob pattern `pep-{}.*` and selecting the extension from that?
Because if the documentation system goes to `.md` or whatever new extension, the website code would again have to be changed here!

Reply:
I'm pretty sure we aren't going to support another documentation format (especially Markdown) within the Python organization. Plus, we are going to move all PEP infrastructure to RtD so YAGNI :)

Reply:
@berkerpeksag OK

Reply:
And if we ever did, we'd probably move generation of the source link inside `pep2html.py` in the PEP's repo (we were even considering that as an option this time).

Reply:
OK @ncoghlan, I see.